src/test/ensemble_tests.cpp

```diff
@@ -1460,6 +1460,22 @@ TEST_F(EnsembleFlowTest, PipelineFactoryWrongConfiguration_NodeNameDuplicate) {
     ASSERT_EQ(factory.createDefinition("pipeline", info, {}, managerWithDummyModel), StatusCode::PIPELINE_NODE_NAME_DUPLICATE);
 }
 
+TEST_F(EnsembleFlowTest, PipelineFactoryWrongConfiguration_PipelineNameOccupiedByModel) {
```

Comment:
I am not sure if full integration test wouldn't be better here (as in line 2719)
https://github.com/openvinotoolkit/model_server/pull/395/files#diff-351b765cae5920e333ff1ced3d998e6aadbbc18b6712b1e7d95f459d2ea34108R2719